helix-rest/src/main/java/org/apache/helix/rest/server/ServerContext.java

```diff
@@ -47,8 +47,8 @@
   private ZKHelixAdmin _zkHelixAdmin;
   private ClusterSetup _clusterSetup;
   private ConfigAccessor _configAccessor;
-  // The lazy initialized base data accessor that reads/writes byte array to ZK
-  private ZkBaseDataAccessor<byte[]> _byteArrayBaseDataAccessor;
+  // A lazily-initialized base data accessor that reads/writes byte array to ZK
```

Comment:
Is it only supporting read at this version? If so, can we update the name accordingly? _byteArrayZkReadOnlyAccessor?

Reply:
Since the comment here does not match what the accessor can do, can we at least comment or add a TODO here?

Reply:
Will add.